### ISSUE TITLE
📄 Consulting Page - CMS

### ISSUE THREAD

**Issue by wicksipedia** [SSWConsulting/SSW.Website#362]
Current page https://www.ssw.com.au/ssw/Consulting/ContentManagement.aspx

**Comment by wicksipedia** [comment]
Parked - waiting on new content from @UlyssesMaclaren

**Comment by UlyssesMaclaren** [comment]
Hey Matt,

Here’s the content:

Want to know the best CMS solutions for your website?
We can help you choose

<Book a free initial meeting section>

The benefits of using a CMS:

Content Management Systems allow for anyone in your organization to publish content painlessly. Whether for your public website, press releases, or internal business portals, content can be drafted as easily as writing an email. Enterprise capabilities such as version control, and workflow, mean the right content gets delivered at the right time, in keeping with company policies.

<image> Easy Content Management: One of the primary benefits of using a CMS is that it allows users to create and manage website content easily, without requiring any technical expertise. CMS users can create new pages, upload images and videos, and publish content without having to write any code or rely on IT support.

<image> Improved Workflow: CMS frameworks often come with tools to manage content workflow, including content approval processes and version control. These features make it easier for teams to collaborate on content creation and ensure that the right content is published at the right time.

<image> Increased Efficiency: Using a CMS can save time and increase efficiency by allowing users to reuse content across multiple pages, automate publishing processes, and easily update content as needed.

<image> Customization and Scalability: CMS frameworks offer a high degree of customization and scalability, allowing users to create custom templates, modules, and plugins to suit their specific needs. This makes it easier to expand and grow a website or content repository as the organization's needs change over time.

Learn more on Do you know the best CMS solutions for websites?<https://www.ssw.com.au/rules/do-you-know-the-best-cms-solutions-for-websites/>

<testimonial section. Etc>

Technologies - Enterprise


  1.  SharePoint: A Microsoft product that is an all-in-one platform for intranet, content management, and collaboration, with strong integration with Microsoft Office tools.
  2.  Kentico: An all-in-one platform for web content management, e-commerce, and online marketing, with a user-friendly interface and flexible customization options.
  3.  Sitecore: A leading enterprise-level CMS with advanced personalization features, multi-channel content management, and analytics capabilities.

Technologies – Medium Businesses


  1.  Sitefinity: A modern and flexible CMS platform with powerful marketing and personalization features, built-in developer tools, and support for multiple languages and sites.
  2.  Jamstack: A modern architecture for building fast, scalable, and secure websites using static site generators, serverless functions, and content APIs.

Technologies – Small Businesses


  1.  DotNetNuke: An open-source CMS built on the Microsoft .NET framework, with a strong developer community, a wide range of modules, and support for multiple sites and languages.
  2.  Umbraco: A flexible and scalable open-source CMS built on the Microsoft .NET framework, with a user-friendly interface and a large developer community.
  3.  Orchard: An open-source CMS built on the Microsoft .NET framework, with a modular architecture, strong content management features, and support for multiple themes and modules.
  4.  WordPress: The most popular open-source CMS for websites and blogs, with a user-friendly interface, extensive plugin and theme library, and a large developer community.











Ulysses Maclaren   |   SSW General Manager<http://www.ssw.com.au/People/Ulysses-Maclaren?utm_source=emailsignature>   |    +61 419 418 232<https://www.ssw.com.au/ssw/Company/Offices/?utm_source=emailsignature>   |   Ping me on ***@***.***>
